### ISSUE TITLE
[FEATURE] Different Style for TOC

### ISSUE THREAD

**Issue by HEIGE-PCloud** [HEIGE-PCloud/DoIt#23]
The `max-width: 800px` setting on the article element leaves a lot of blank space on a wide screen. The position and margin of TOC and article need some redesigns.

Different styles can be left as a setting which can be switched by the user manually for each post or entire site, or it can be adjusted by the reader with a button.

**Comment by HEIGE-PCloud** [comment]
## Possible layout 1

![image](https://user-images.githubusercontent.com/52968553/116002210-88282b00-a5f0-11eb-8191-616dbde3a21f.png)

```css
.page {
    margin-left: 10%;
    margin-right: auto;
    max-width: initial;
    width: 70%;
}
```

**Comment by HEIGE-PCloud** [comment]
## Possible layout 2

![image](https://user-images.githubusercontent.com/52968553/116003323-60879180-a5f5-11eb-97bf-8c8500b5fed9.png)

```css
.page {
    margin-left: 324px;
    margin-right: auto;
    max-width: initial;
    width: 70%;
}
.toc-auto {
    right: 1168px;
    max-width: 328px;
    visibility: visible;
    position: fixed;
    top: 76px;
}
```

**Comment by HEIGE-PCloud** [comment]
## Possible layout 3

![image](https://user-images.githubusercontent.com/52968553/116003782-69796280-a5f7-11eb-9e50-1b2310196f1b.png)

![image](https://user-images.githubusercontent.com/52968553/116003812-9299f300-a5f7-11eb-8750-97cdcca0eb71.png)

Use a button to turn TOC on and off. The CSS needs to be further optimized.

**Comment by HEIGE-PCloud** [comment]
I think change `max-width: 800px;` to `max-width: initial;` can be the first harmless step.

**Comment by wu-yue-yu** [comment]
Wow, that's what I'm looking for, I like LoveIt theme, but it seems that the author has abandoned it.
I always thinks that the width is too small, so the reading experience is not good.
BTW, I prefer layouts 1 :)

**Comment by HEIGE-PCloud** [comment]
Layout 1 has been implemented. This issue will be closed by #35 , if there are future needs for other possible layouts, it's more than welcome to reopen this issue.

**Comment by HEIGE-PCloud** [comment]
## Possible layout 4

Add a button to switch toc between `auto-toc` and `static-toc`. Should be quite easy to implement.
